src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -534,6 +540,11 @@ def __init__(self):
         self._register_default_ciphers()
         self.activate_osrandom_engine()
 
+    def openssl_assert(self, ok):
+        if not ok:
+            errors = self._consume_errors()
+            raise UnhandledOpenSSLError("Unknown OpenSSL error", errors)
```

Comment:
That seems reasonable. "Unknown OpenSSL error. Please file an issue at https://github.com/pyca/cryptography/issues with information on how to reproduce" sound good?